daemon/cmd/local_node_sync.go

```diff
@@ -184,8 +184,16 @@ func (ini *localNodeSynchronizer) initFromK8s(ctx context.Context, node *node.Lo
 				node.SetCiliumInternalIP(net.ParseIP(addr.IP))
 			}
 		}
+
+		if ini.Config.EnableIPv4 {
+			node.IPv4HealthIP = net.ParseIP(k8sCiliumNode.Spec.HealthAddressing.IPv4)
+		}
+
+		if ini.Config.EnableIPv6 {
+			node.IPv6HealthIP = net.ParseIP(k8sCiliumNode.Spec.HealthAddressing.IPv6)
+		}
```

Comment:
Sorry, forgot to mention that you should also check for `ini.Config.EnableHealthChecking && ini.Config.EnableEndpointHealthChecking` for consistency with the `allocateHealthIPs` check.

Reply:
Done